homeassistant/components/upnp/config_flow.py

```diff
@@ -220,7 +223,7 @@ async def async_step_ssdp(self, discovery_info: Mapping) -> Mapping[str, Any]:
             return self.async_abort(reason="incomplete_discovery")
 
         # Ensure not already configuring/configured.
-        unique_id = discovery_info[ssdp.ATTR_SSDP_USN]
+        unique_id = discovery_info.ssdp_usn
         await self.async_set_unique_id(unique_id)
         hostname = discovery_info["_host"]
```

Comment:
Should we make this `discovery_info.ssdp_headers["_host"]`?

Reply:
`discovery_info.ssdp_headers["_host"]` or `discovery_info.upnp["_host"]`?
That's what I asked in my earlier comment.

Reply:
It gets injected here  https://github.com/StevenLooman/async_upnp_client/blob/development/async_upnp_client/ssdp.py#L141 so I think `ssdp_headers`

Reply:
`nanoleaf` uses ssdp_headers, so I've lined up with that